src/trustyai/metrics/saliency.py

```diff
@@ -0,0 +1,136 @@
+# pylint: disable = import-error
+"""Saliency evaluation metrics"""
+from org.apache.commons.lang3.tuple import (
+    Pair as _Pair,
+)
+
+from trustyai.model import simple_prediction
+from trustyai.explainers.lime import LimeExplainer
+from trustyai.explainers.shap import SHAPExplainer
+from trustyai.explainers import LocalExplainer
+
+from . import ExplainabilityMetrics
+
+
+def impact_score(model, pred_input, explainer, k, is_model_callable=False):
```

Comment:
Perhaps we could add argument typing in these signatures.

Reply:
sure